### PR TITLE
fixed some broken links

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Fixes some of the broken links.
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0ff00b50bda1d1391f5ea6cf12da7313a45cf5f3 at: https://610d03d84b304990394b71e5--layer5ng.netlify.app

File: src/collections/service-mesh-workshops/2020_05_12_open_source_101/index.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "Open Source 101 at Home"
 abstract: "Solving the Service Mesh Adopter’s Dilemma - Open Source 101 at Home 2020"
 thumbnail: "./open_source_101_original.jpg"
-eurl: https://opensource101.com/sessions/solving-the-service-mesh-adopters-dilemma/
+
```
Comment (Rubix982) [inline src/collections/service-mesh-workshops/2020_05_12_open_source_101/index.mdx:5]: Should we place the `https://opensource101.com/` URL with the YT substitute? It is 'https://www.youtube.com/watch?v=Q1zSWbO0RmI'

**Review by Rubix982 — CHANGES_REQUESTED** [review]
Everything looks great! Wanted feedback over the `eurl`.

**Comment by warunicorn19** [comment]
Yeah we could do that, thanks @Rubix982. Some more links needs to be fixed that's why I've put the pr on hold.

File: src/collections/service-mesh-workshops/2020_05_12_open_source_101/index.mdx (hunk 1 above)
Comment (Nikhil-Ladha) [reply src/collections/service-mesh-workshops/2020_05_12_open_source_101/index.mdx:5]: Don't add a blank line instead.

**Review by Jashpatel1 — APPROVED** [review]

**Pull request merged**